react-app/src/components/common/Badge/Badge.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import cn from "classnames";
+
+type BadgeVariant = "light" | "medium";
+
+type BadgeColor = "yellow";
+
+const ColorClasses = {
+  "medium-yellow": ["bg-likecoin-yellow"],
+  "light-yellow": ["bg-yellow-100", "text-yellow-800"],
+};
```

Comment:
Would we need to implement both variants for each color? i.e if I was to add a `red` here we would need `medium-red` and `light-red`? how about we make `likecoin-yellow` part of the BadgeColor type instead?

Reply:
> Also I think we would want the user to specify a color when using this component so maybe we can may the color prop mandatory instead of giving it a default value?

Yes let me remove the default value



> Would we need to implement both variants for each color? i.e if I was to add a `red` here we would need `medium-red` and `light-red`? how about we make `likecoin-yellow` part of the BadgeColor type instead?

So rather than having both the color and variant prop, we would have only the color prop instead?

Reply:
yes i think that would be the simplest way to handle this case. Please also specify the type for the ColorClasses variable